layer.yaml

```diff
@@ -15,6 +15,11 @@ options:
     puppet-version: '3'
     silent: True
 defines:
+  install_java:
+    type: string
+    default: 'openjdk-8-jdk'
```

Comment:
@johnsca People thought that the packages might need some of the tools in the development kit. We can switch it over to just the runtime environment if you'd rather.